spotify

```diff
@@ -26,6 +26,35 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+USER_CONFIG_DEFAULTS='CLIENT_ID=""
+CLIENT_SECRET=""'
+USER_CONFIG_FILE="${HOME}/.shpotify.cfg"
+if ! [[ -f "${USER_CONFIG_FILE}" ]]; then
+    touch "${USER_CONFIG_FILE}"
+    echo "${USER_CONFIG_DEFAULTS}" > "${USER_CONFIG_FILE}"
+fi
+source "${USER_CONFIG_FILE}"
+
+showAPIHelp() {
```

Comment:
Just spit-balling here, feel free to tear this apart

Reply:
Also just noticed your `;`'s -- I'll go back and add those in to stay consistent when I get some more time to work on this